specs/login.spec

```diff
@@ -1,6 +1,6 @@
 # Login in amazon
-the user can login in the amazon account
+Automated testing to check if user logged in successfully
 
 ## Successful Login
 
-* The required username is "9654081639"
+* Open the amazon account of "9654081639"
```

Comment:
This step is not understandable. As this step does not justify that you are visiting to Amanon.com and entering the credentials and then verifying the login page .

Maybe something like this ( just an example)


- Go to the amazon login page
- Enter "userName" as "9654081639"
- Enter "password"
- Click on the "Sign" button

Reply:
These are generic steps that are taking parameters. Please also check this link https://docs.gauge.org/writing-specifications.html?os=macos&language=javascript&ide=vscode#step-alias